src/zcash/ZcashEngine.ts

```diff
@@ -72,15 +73,9 @@ export class ZcashEngine extends CurrencyEngine<
   }
 
   initData(): void {
-    const { birthdayHeight } = this.initializer
-
     // walletLocalData
-    if (this.otherData.blockRange.first === 0) {
-      this.otherData.blockRange = {
-        first: birthdayHeight,
-        last: birthdayHeight
-      }
-    }
+    this.otherData.isSdkInitializedOnDisk = true
```

Comment:
Shouldn't this default to false, to match the cleaner?

Reply:
The purpose of this field is to track if we've ever initialized a synchronizer on this device before. The flag should be false the very first time the synchronizer is started but true for every other synchronizer start (including across resyncs where the zcash dbs are still present on disk). 

So it defaults to false the first time the synchronizer starts but will always be true after that.